app/src/Misc/MetaData.php

```diff
@@ -6,23 +6,19 @@ class MetaData
 {
     public const TEAMS = [
         'product' => [
-            'maxime-rainville',
             'emteknetnz',
             'GuySartorelli',
-            'sabina-talipova',
         ],
         'core-committers' => [
             'sminnee',
             'wilr',
             'chillu',
+            'maxime-rainville',
             'michalkleiner',
             'Cheddam',
-            'dhensby',
             'unclecheese',
             'madmatt',
             'kinglozzer',
-            'ScopeyNZ',
-            'Cheddam',
```

Comment:
Cheddam (Garion) is still a core committer

Reply:
He's also still in the list ;p There was a duplicate.